app/config/override.yaml

```diff
@@ -0,0 +1,12 @@
+cert-exporter:
+  chart: "cert-exporter",
+# chart-operator must be installed first so the chart CRD is
+# created in the tenant cluster.
+chart-operator:
+  chart:     "chart-operator",
+  namespace: "giantswarm",
+# CoreDNS's Upgrade force is disabled to avoid affecting customer workloads.
+coredns:
+  useUpgradeForce: false,
+net-exporter:
+  chart: "net-exporter",
```

Comment:
Do we need `net-exporter` and `cert-exporter` in here?

You can also get rid of trailing commas.

Reply:
Only these two apps don't have `%s-app` suffix in their chart-name. I also would like to change this, but since we need to think about the compatibility between older releases, I would like to keep `chart` in here.

Reply:
Ack.